query/tests/it/sql/optimizer/heuristic/testdata/select.test

```diff
@@ -62,7 +62,7 @@ Scan: default.system.numbers
 select a from (select number as a, number+1 as b from numbers(1))
 ----
 Project: [number (#0)]
-    EvalScalar: [+(numbers.number (#0), 1)]
+    EvalScalar: []
```

Comment:
Seems after `post_optimize`, there are still possibilities for optimization.

We can consider in the future.

Reply:
Yes